5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
-# Changes compared to v4.x
+# Changes Compared to v4.x
 
 ## Introduction
 
-Existing users of the 4.x version of the standard may find it useful to see some of the key changes in the content of version 5.0 as well some changes in scope and philosophy that took place during the development of version 5.0.
+Users familiar with version 4.x of the standard may find it helpful to review the key changes introduced in version 5.0, including updates in content, scope, and underlying philosophy.
 
-Overall, only 11 out of 286 requirements from version 4.0.3 have stayed as they were and 15 requirements had grammar changes but no change to the requirement's meaning. The rest have been modified in some way. Even those requirements that were not modified or moved will have numbering changes as a result of reordering or other requirements moving around.
+Of the 286 requirements in version 4.0.3, only 11 remain unchanged, while 15 have undergone minor grammatical adjustments without altering their meaning. The majority have been revised in some way. Even requirements that were not substantively modified may have new numbering due to reordering or restructuring.
```

Comment:
Maybe interesting to use for context - 50 requirements where just deleted. Additional 28 requirements were deleted with "COVERED BY" which de facto means "duplicates", 31 requirements merged into other. Total 109 requirements were deleted... which is 38%.

Reply:
> substantively modified may have new numbering

"substantively modified ~~may~~ have new numbering". There is no "may".

Reply:
Making modifications along these lines